.gitmodules

```diff
@@ -1,29 +1,32 @@
 [submodule "components/clm/src/external_models/fates"]
-	path = components/clm/src/external_models/fates
```

Comment:
These whitespace changes will cause conflicts if the upstream changes parts of this file. IMO, it's best just to add your submodule and leave everything else as is.

Reply:
Ah, it must have been my vim, changing tabs into spaces. But this does not explain why test-all would pass when I manually run it.

Reply:
No, I mean the true upstream, E3SM. When we do periodic upstream/E3SM merges, it's good not to have to resolve conflicts in infrastructure files due to whitespace changes.

Reply:
Ah, I see. I will push a fix for this.